### ISSUE TITLE
Key Maps/Conflicts to Buffer Not Working on Sublime Text 3

### ISSUE THREAD

**Issue by halhenke** [skuroda/FindKeyConflicts#5]
Build 3065

Cant remember if it was working before on ST 3 or if it was only on ST 2.

Sending output to Quick Panel works.

**Comment by KeyserS0ze** [comment]
I am running Build 3103. I am getting a similar problem running FindKeyConflicts.

I run FindKeyConflicts: All Conflicts, it runs for a short while (shown in the status bar), then quickly shows the message "FindKeyConflicts Finished", but nothing shows up. It could be that at some point I have seen a new tab created with the output. The following appears in the console:

Traceback (most recent call last):
  File "find_key_conflicts in C:\SublimeText3\Data\Installed Packages\FindKeyConflicts.sublime-package", line 82, in <lambda>
  File "find_key_conflicts in C:\SublimeText3\Data\Installed Packages\FindKeyConflicts.sublime-package", line 97, in handle_thread
  File "find_key_conflicts in C:\SublimeText3\Data\Installed Packages\FindKeyConflicts.sublime-package", line 255, in handle_results
  File "find_key_conflicts in C:\SublimeText3\Data\Installed Packages\FindKeyConflicts.sublime-package", line 166, in generate_key_map_text
  File "find_key_conflicts in C:\SublimeText3\Data\Installed Packages\FindKeyConflicts.sublime-package", line 196, in generate_text
KeyError: 'command'

Additionally, "(Direct) Conflicts to Buffer", and "All Key Maps to Buffer" do not work. All the other commands do provide an output.

**Comment by divinites** [comment]
The same problem as KeyserS0ze has.

**Comment by skuroda** [comment]
@KeyserS0ze @divinites apologies I thought I had responded to this. Can you tell me what version of the plugin you are using (visible by doing `Package Control: List Packages` if you installed via package control, or the hash for the version you have checked out.  

Thanks.

**Comment by divinites** [comment]
Thanks for your quick reply.  I use the following version: v2016.03.28.05.41.59

**Comment by KeyserS0ze** [comment]
Dear skuroda,

No worries. Thanks for your plugins! They are very helpful! And thanks for making them free, and open source! This is true evolution: everyone helping, improving, sharing, and benefiting from that! Good on'ya!

The version that I use is: v2016.03.28.05.41.59. ST: v3 3103.

It is annoying, extremely unhelpful, but I must confess: after that comment, it did work some other day. I tested, without great expectations, but still, it worked. The intermittency kills me. Out of love for the Scientific Method, I have just ran in the last minute:

"caption": "FindKeyConflicts: (Direct) Conflicts to Buffer",
"caption": "FindKeyConflicts: All Key Maps to Buffer",
"caption": "FindKeyConflicts: All Conflicts",
"caption": "FindKeyConflicts: Overlap Conflicts",
"caption": "FindKeyConflicts: Single Package Conflicts",

And all of them created a new tab, with the appropriate information, perfectly, how it was supposed to work. It is great that it worked, and it seems to be working consistently, but the intermittency bothers me. I really cannot reproduce the error that happened in the past. In fact, I have no clue what caused it, or what cured it.

It could be that I have removed one or two packages, or that I installed another one or two, between my report of the problem, and the first positive result, hinting conflict of packages perhaps? Right now, I also have Keymaps installed, and also working perfectly.

Thus, right now, I cannot complain of any problem. To me, all is fine, as expected. Would there be anything else that I could do to help to find the problem?

**Comment by divinites** [comment]
After reading KeyserS0ze's reply, I retry the plug-in again, it also works...  I did not do anything special actually, just wake up the macbookair as usual. However, I restart my laptop several time last night, it did not work.

**Comment by skuroda** [comment]
Well good news, a quick look, and I think I've found the bug. Perhaps one of the plugins you had or keymaps you defined created a key mapping without an associated command. e..g

```
{
    "keys": ["f8"]
}
```

in one of the keymap files. I'll add a warning for keymaps missing command files.

**Comment by skuroda** [comment]
I've pushed the fix, and will close the issue. Of course, if you see it again, please do not hesitate to tell me. As always, please include the version of ST, the version of the plugin, as well as the stack trace if possible.

Thank you for using FindKeyConflicts.

**Comment by KeyserS0ze** [comment]
Hey skuroda, thanks for your work. I will keep an eye on this symptom. Now that you mentioned this mapping, I must confess that previously, I had mapped the standard multiple cursors binding to simply nothing, because of the conflict with my video driver's binding for flipping the screen. Then, later on, I remapped it to something more purposeful and beautiful, like:
    { "keys": ["ctrl+alt+up"], "command": "null" },
    { "keys": ["ctrl+alt+down"], "command": "null" },

I just run all commands, and all seems fine. It is great to see that this guy is still being looked after. If it misbehaves, I certainly will let you know.
Good on'ya.
ks
